Newspack/Newspack/Models/StoryAsset+CoreDataClass.swift

```diff
@@ -2,15 +2,18 @@ import Foundation
 import CoreData
 
 @objc(StoryAsset)
-public class StoryAsset: NSManagedObject {
+public class StoryAsset: NSManagedObject, TextNoteCellProvider, PhotoCellProvider, VideoCellProvider, AudioCellProvider {
+
+    var caption: String! {
```

Comment:
Maybe drop the `!` ?

Reply:
Hmm...yeah.  I did that for consistency with other provider properties that basically just reference the StoryAsset properties. I'd really prefer getting rid of the optionals altogether. I'll give this some more thought and revisit.